core/settings.py

```diff
@@ -11,9 +11,8 @@
 SECRET_KEY = config('SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = config('DEBUG', default=False, cast=bool)
+DEBUG = config('DEBUG', default=True, cast=bool)
```

Comment:
@rodgomesc I don't think this is a good idea. On a deploy if there is no enviroment variables the application will go to production with DEBUG mode ON.

Reply:
sorry, I changed this for testing and forgot to update